src/dino_ip_adapter_refiners/mixin/ip_adapter.py

```diff
@@ -104,44 +255,73 @@ def enable_gradients(self, enable: bool) -> None:
         self.key_matrix.weight.requires_grad_(enable)
         self.value_matrix.weight.requires_grad_(enable)
 
-class DinoOnlyIPAdapter(Adapter[SD1UNet], fl.Chain):
-    def __init__(self, target: SD1UNet, use_timestep_embedding: bool = False, use_uncond_token: bool = True) -> None:
+class UncondImageEmbedding(fl.Module):
```

Comment:
oh you can do that? I was trying to but I thought with the chain you can't do that

Reply:
You cannot register a Module, but you can register a parameter. The reason you can't register modules is that it will mess with the forward of the Chain (since it chain's over the list of its registered submodules)

Reply:
makes perfect sense. Fixed